src/main/kotlin/com/progreizh/animaliste/controllers/AccountController.kt

```diff
@@ -0,0 +1,84 @@
+package com.progreizh.animaliste.controllers
+
+import com.progreizh.animaliste.entities.Account
+import com.progreizh.animaliste.entities.AccountLoginForm
+import com.progreizh.animaliste.repositories.AccountRepository
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import java.util.Optional
+
+@RestController
+@RequestMapping("/accounts")
+class AccountController(private val repository: AccountRepository) {
+    /**
+     * Récupère tous les comptes.
+     */
+    @GetMapping
+    fun getAllAccounts(): ResponseEntity<List<Account>>{
+        val accounts = repository.findAll()
+        return ResponseEntity.ok(accounts)
+    }
+
+    /**
+     * Récupère le compte en fonction de son identifiant.
+     */
+    @GetMapping("/{id}")
+    fun getOneAccountById(@PathVariable("id") id: String): ResponseEntity<Account> {
```

Comment:
```suggestion
    fun getAccount(@PathVariable("id") id: String): ResponseEntity<Account> {
```
Le prototype de la méthode est assez explicite. Pas besoin de dire byId. De plus , by id est la maniere de base.